BREAKING.md

```diff
@@ -35,6 +35,19 @@ to be `string | undefined`.
 ### `IContainerRuntime.flush` is deprecated
 `IContainerRuntime.flush` is deprecated and will be removed in a future release. If a more manual flushing process is needed, move all usage to `IContainerRuntimeBase.orderSequentially` if possible.
 
+# 1.2.0
+
+## 1.2.0 Upcoming changes
+- [ Added locationRedirection errorType in DriverErrorType enum](#Added-locationRedirection-errorType-in-DriverErrorType-enum)
+- [ Added ILocationRedirectionError error in DriverError type](#Added-ILocationRedirectionError-error-in-DriverError-type)
+
+ ### Added locationRedirection errorType in DriverErrorType enum
+ Added locationRedirection errorType in DriverErrorType enum. This error tells that the location of file on server has changed.
+ This error will not be thrown in 1.x.x version but we are just adding it in the type for now. This will be thrown from 2.x.x onward.
```

Comment:
I think it should talk action. I.e. what you want customers to do - they need to add handling of these errors now, especially in cases where solutions use dynamically loaded driver.

Basically, if I know nothing about this feature (I just joined some team and bump OCE), I need to know what I need to do here.